packages/comet/src/logger.ts

```diff
@@ -27,6 +27,8 @@ export const logger: Logger = {
 }
 
 export function recordException(exception: unknown) {
+  console.error('[Comet] Recorded exception', exception)
```

Comment:
Do we log to the console anywhere else?

Reply:
Hmm, it's all channeled into otel logs now.. should I remove this one too?